app/src/main/java/ch/hikemate/app/model/route/HikeRoute.kt

```diff
@@ -0,0 +1,47 @@
+package ch.hikemate.app.model.route
+
+/** A class representing a bounding box */
+data class Bounds(val minLat: Double, val minLon: Double, val maxLat: Double, val maxLon: Double) {
+  class Builder {
```

Comment:
Nitpick: This function is very well thought with the use of the `Builder`, but it would be good to check general conditions like minLat<=maxLat and such just to make sure the `Bounds` are well defined.

Reply:
Great idea! Implemented it!